src/aria/Aria.js

```diff
@@ -821,6 +821,21 @@ var Aria = module.exports = global.Aria;
         return res;
     };
 
+    /**
+     * Returns the classpath of the given reference, if it is not already a classpath.
+     * @param {String|Object} classpathOrCstr Classpath or reference to a class.
+     */
+    Aria.getClasspath = function (classpathOrCstr) {
```

Comment:
Misses a comment about the case where the input wouldn't be a string and the classpath couldn't be detected. Depending on the desired behavior, according code might miss as well.